opencti-platform/opencti-front/src/components/filters/FilterChipPopover.tsx

```diff
@@ -223,25 +226,54 @@ export const FilterChipPopover: FunctionComponent<FilterChipMenuProps> = ({
       availableRelationFilterTypes={availableRelationFilterTypes}
     />
   );
+  const getSelectedOptions = (entitiesOptions: OptionValue[]): OptionValue[] => {
+    // we try to get first the element from the search
+    // and if we did not find we tried one from filterReprensentative
+    // Most of the time element from search should be sufficient
+    const mapFilterValues: OptionValue[] = [];
+    filterValues.forEach((value: string) => {
+      const mapRepresentative = entitiesOptions.find((f) => f.value === value);
+      if (mapRepresentative) {
+        mapFilterValues.push({
+          ...mapRepresentative,
+          group: t_i18n('selected'),
+        });
+      } else {
+        const filterRepresentative = filtersRepresentativesMap.get(value);
+        if (filterRepresentative) {
+          mapFilterValues.push({
+            value,
+            type: filterRepresentative?.entity_type || t_i18n('Unknown'),
+            parentTypes: [],
+            group: t_i18n('selected'),
+            label: filterRepresentative?.value ?? t_i18n('Unknown'),
+            color: filterRepresentative?.color ?? undefined,
+          });
+        }
+      }
+    });
+    return mapFilterValues.sort((a, b) => a.label.localeCompare(b.label));
+  };
 
   const buildAutocompleteFilter = (fKey: string, subKey?: string): ReactNode => {
+    const getOptions = getOptionsFromEntities(entities, searchScope, fKey);
+    const entitiesOptions = getOptions
+      .filter((option) => !filterValues.includes(option.value));
+
+    const selectedOptions: OptionValue[] = getSelectedOptions(getOptions);
+
+    const groupByEntities = (option: OptionValue) => {
+      return t_i18n(option?.group ? option?.group : fKey);
+    };
     return (
-      <MUIAutocomplete
-        disableCloseOnSelect
+      <Autocomplete
+        multiple
         key={fKey}
-        selectOnFocus={true}
-        autoSelect={false}
-        autoHighlight={true}
         getOptionLabel={(option) => option.label ?? ''}
         noOptionsText={t_i18n('No available options')}
-        options={getOptionsFromEntities(entities, searchScope, fKey)}
-        groupBy={
```

Comment:
Remove into a function but remove also the checking if isStixObjecTypes. In the searchEntitesUtil, we do a mapping where if `group `is undefined we use the `type `value.